src/internal/operations/backup.go

```diff
@@ -573,10 +573,11 @@ func mergeDetails(
 				newPath.String(),
 				newPath.ShortRef(),
 				newPath.ToBuilder().Dir().ShortRef(),
+				"", // TODO Location Ref,
```

Comment:
we need to fix this before #2429 merges. Otherwise we'll break incremental backups for calendars